template/tests/jest.setup.js

```diff
@@ -13,9 +13,14 @@ module.exports = async () => {
   //   isUp = await statusCheck();
   // }
   // console.log(`[jest] server is up...`);
+  const testDatabaseName = 'testing';
+  const testSchema = `test_${nanoid().toLowerCase()}`;
+  const testDatabaseUrl = new URL(process.env.DATABASE_URL);
+  testDatabaseUrl.pathname = `/${testDatabaseName}`;
```

Comment:
Do we need to modify the database used @cball? Or will the different schema suffice?

Reply:
The only potential downside I can think of here is if someone creates a postgres user that is not a super user or does not have `create schema` permissions.

We either need to require that the postgres user has the right permissions, or we should take `DATABASE_URL` wholesale if defined.

I'm leaning towards requiring a postgres user with the right permissions and keeping the auto generated schema. what do you think?